lessons/AI/ai-decision-trees/langs/en-us/index.adoc

```diff
@@ -0,0 +1,358 @@
+
+@QandA{
+@Q{With your partner, look over the @handout{decision-tree-data.adoc, Training Dataset}. What do you Notice? What do you wonder?}
+@A{Possible responses:}
+@A{Individuals in their twenties always buy the video game.} 
+@A{There are only three new customers; two out of three times, new customers buy the video game.}
+@Q{Can you foresee any problems with making a decision tree based on this dataset? If so, what are they?}
+@A{Responses will vary.}
+}
+
+One problem with this dataset is that _age is continuous_. That won’t work!
+
+We need to break these ages down into different groups - which will become different _branches_ that grow out of a _decision node_. *For now, let’s agree to create three groups: teenagers, twenties, and thirties.*
+
+We can use a different representation, called a @vocab{tree stump} for this than we did with the yes-no questions we saw in our previous decision trees.
+
+
+@lesson-instruction{
+- We will complete @printable-exercise{level-1.adoc} together, starting with "age" as the root node.
+- As we create the first level of our tree, you will discover a tool that we use to complete all the levels of our model: the decision stump.
+}
+
+@teacher{
+A @vocab{decision stump} is a one-level decision tree that makes a prediction based on the value of just a single input feature.
+
+Students are introduced to the @vocab{decision stump} in question 4 on @printable-exercise{level-1.adoc}. Be prepared to guide students through building the decision stump. They will need to use (1) the rule that you wrote in question 1 and (2) the training dataset.
+
+- The stump has three branches because we are considering customers in their teens, twenties and thirties.
+- The left-most leaf node ("teens") represents the five teens in our training dataset: Jan (16), Jose (19), Jillian (14), Ariella (16), and Danial (19).
+** Jan, Jose, and Jillian did *not* purchase the game, so they are represented by the letter N (for "no").
+** Ariella and Danial *did* purchase the game, so they are represented by the letter Y (for "yes").
+** We illustrate the teens' decisions with the following shorthand: N N N Y Y
+- Finally, we place checkmarks below correctly predicted values to indicate how successful our rule was.
+** Our rule predicted that individuals in their teens would *not* purchase the game, so we place checkmarks by the Ns that represent Jan, Jose, and Jillian. Our rule was correct.
+** We leave the Ys *without* checkmarks; our rule was wrong for Danial and Ariella.
+- We complete the same process for customers in their twenties and in their thirties, representing each individual in the training dataset with a "Y" or "N", and then placing checkmarks to indicate when our rule was successful.
+}
+
```

Comment:
We need to teach about what "pure" in the lesson plan, not just on the workbook page.